configure.ac

```diff
@@ -72,6 +71,33 @@ AC_ARG_ENABLE(
 AS_IF([test "x$esapi_sf" = "xyes"],
        [do_esapi_manage_flags])
 
+AC_ARG_WITH(
+  [fapi],
+  [AS_HELP_STRING([--with-fapi],
+    [enable or disable the fapi backend. Default is "auto" to autodetect])],
+    [with_fapi=$enableval],
+    [with_fapi=auto])
+
+AS_IF([test "x$enable_overflow" = "xno"],
```

Comment:
This part I don't understand ?
Different workstream ?

Reply:
me either, I think it's just a copy paste error.